components/bulletins/BulletinCard.vue

```diff
@@ -27,22 +27,22 @@
 </template>
 
 <script>
-import TextButton from '@/components/core/buttons/TextButton'
+// import TextButton from '@/components/core/buttons/TextButton'
 
 export default {
     name: 'BulletinCard',
     components: {
-        TextButton,
+        // TextButton,
     },
     props: {
         title: { type: String, default: '' },
         description: {
             type: String,
             default: '',
         },
-        linkTitle: { type: String, default: '' },
-        linkHref: { type: String, default: '' },
-        linkTo: { type: String, default: '' },
+        // linkTitle: { type: String, default: '' },
+        // linkHref: { type: String, default: '' },
+        // linkTo: { type: String, default: '' },
```

Comment:
These comments will break the use of `<text-button>` and probably break the flexibility of this card.  It is better to have an option like [here](https://github.com/pycontw/pycontw-frontend/blob/main/components/sponsors/SponsorCard.vue#L5) so that other developers could decide if to use the text button instead of removing it directly.

Reply:
Uncommented.